src/test/java/com/powsybl/openloadflow/ac/AcLoadFlowVscTest.java

```diff
@@ -207,32 +207,6 @@ void testHvdcAcEmulationNonSupported() {
         assertReactivePowerEquals(-10.0, cs3.getTerminal());
     }
 
-    @Test
-    void testHvdcAcEmulationNonSupported2() {
```

Comment:
You don't want to keep it with a non zero P0 maybe?

Reply:
I do not understand @annetill . non zero P0 is covered in testDcLoadFlowWithHvdcAcEmulation2 as well.
DId I remove a test that is not performed in the new one  ?

Reply:
Yes sorry, I see it now.